src/spectrecoff-cli/commands/BreakdownChart.fs

```diff
@@ -0,0 +1,51 @@
+﻿namespace SpectreCoff.Cli.Commands
+
+open Spectre.Console.Cli
+open Spectre.Console
+open SpectreCoff.Cli
+open SpectreCoff.Output
+open SpectreCoff.Rule
+open SpectreCoff.Layout
+open SpectreCoff.Chart
+open SpectreCoff.Chart.BreakdownChart
+
+type BreakdownChartSettings() =
+    inherit CommandSettings()
+
+type BreakdownChartExample() =
+    inherit Command<BreakdownChartSettings>()
+    interface ICommandLimiter<BreakdownChartSettings>
+
+    override _.Execute(_context, _settings) =
+        Many [
+            NL
+            E "My life as a developer :("
+            NL
+        ] |> toConsole
+
+        let items = [
+            ChartItem ("Refinements", 2)
+            ChartItem ("Dailies", 2)
+            ChartItem ("Retrospectives", 1)
+            ChartItem ("Random meetings", 7)
+            ChartItem ("Fixing bugs", 3)
+            ChartItemWithColor ("Developing features", 1, Color.Red)
+        ]
+
+        breakdownChart items |> toConsole
```

Comment:
Why not prove right away that Many works correctly :)

```
Many [
    NL
    E "My life as a developer :("
    NL
    breakdownChart items
] |> toConsole
```

Reply:
![image](https://user-images.githubusercontent.com/17160319/204149802-130abda2-3eb9-4b90-a709-816e2255a88e.png)

You're right, of course.